src/SonarQube.Client/Services/SonarQubeService.cs

```diff
@@ -228,7 +228,9 @@ public Uri GetProjectDashboardUrl(string projectKey)
         {
             EnsureIsConnected();
 
-            var request = new RoslynExportProfileRequest { QualityProfileName = qualityProfileName, LanguageKey = language.Key };
+            var request = this.serverVersion >= new Version(6, 6)
+                ? new RoslynExportProfileRequestV66Plus { QualityProfileName = qualityProfileName, LanguageKey = language.Key }
+                : new RoslynExportProfileRequest { QualityProfileName = qualityProfileName, LanguageKey = language.Key };
```

Comment:
How hard would it be to add a test for this in _SonarQubeClientTests.cs_?

Reply:
Done